_news/153.md

```diff
@@ -0,0 +1,150 @@
+
+Many thanks to all contributors!
+
+Below are some highlights of the release:
+
+## Highlights
+
+### erts & jit
+- The JIT now works for 64-bit ARM processors.
+- The JIT now does type-based optimizations based on type
+  information in the BEAM files.
+- Improved the JIT's support for external tools like `perf`
+  and `gdb`, allowing them to show line numbers and even
+  the original Erlang source code when that can be found.
+
+### erts, stdlib, kernel
+- Users can now configure ETS tables with the
+  `{write_concurrency, auto}` option. This option forces
+  tables to automatically change the number of locks that
+  are used at run-time depending on how much concurrency
+  is detected. The `{decentralized_counters, true}` option
+  is enabled by default when `{write_concurrency, auto}` is
+  active.
+
+  Benchmark results comparing this option with the other
+  ETS optimization options are available here:
+  [benchmarks](https://erlang.org/bench/ets_bench_result_lock_config.html).
+
+- To enable more optimizations, BEAM files compiled with
+  OTP 21 and earlier cannot be loaded in OTP 25.
+
+- The signal queue of a process with
+  the process flag `message_queue_data=off_heap` has been optimized to
+  allow parallel reception of signals from multiple processes.
+  This can improve performance when many processes are sending in parallel to
+  one process. [See benchmark](https://erlang.org/bench/sigq_bench_result.html).
+
+- The Erlang installation directory is now relocatable on
+  the file system given that the paths in the
+  installation's RELEASES file are paths that are
+  relative to the installations root directory. 
+- A new option called `short` has been added to the
+  functions `erlang:float_to_list` and
+  `erlang:float_to_binary/`. This option creates the
+  shortest correctly rounded string representation of the
+  given float that can be converted back to the same
+  float again.
+- Introduction of `quote/1` and `unquote/1` functions in
+  the `uri_string` module - a replacement for the deprecated functions `http_uri:encode`
+  and `http_uri:decode`.
+- The new module `peer` supersedes the `slave` module. The
+  `slave` module is now deprecated and will be removed in OTP 27.
+
+- `global` will now by default prevent
+  overlapping partitions due to network issues. This is done by
+  actively disconnecting from nodes that reports that
+  they have lost connections to other nodes. This will
+  cause fully connected partitions to form instead of
+  leaving the network in a state with overlapping
+  partitions.
+
+  It is possible to turn off the new behavior by setting the
+  the `kernel` configuration parameter `prevent_overlapping_partitions` to `false`.
+  Doing this will retain the same behavior as in OTP 24 and earlier.
+
+- The `format_status/2` callback for `gen_server`, `gen_statem`
```

Comment:
```suggestion
  Doing this will retain the same behavior as in OTP 24 and earlier.
- The `format_status/2` callback for `gen_server`, `gen_statem`
```